oem/firstvoices/android/app/src/main/assets/setup/main.html

```diff
@@ -1,61 +1,77 @@
-	        }
-
-	        function showRegionList() {
-	            window.jsInterface.showRegionList();
-	        }
-
-	        function setCheckBox1On() {
-	            var checkbox1 = document.getElementById("checkbox1");
-	            if (checkbox1.src != "check-on.png") {
-	                checkbox1.src = "check-on.png";
-	                checkbox1.style.opacity = 1.0;
-	            }
-	        }
-
-	        function setCheckBox1Off() {
-	            var checkbox1 = document.getElementById("checkbox1");
-	            if (checkbox1.src != "check-off.png") {
-	                checkbox1.src = "check-off.png";
-	                checkbox1.style.opacity = 0.25;
-	            }
-	        }
-
-	        function setCheckBox2On() {
-	            var checkbox2 = document.getElementById("checkbox2");
-	            if (checkbox2.src != "check-on.png") {
-	                checkbox2.src = "check-on.png";
-	                checkbox2.style.opacity = 1.0;
-	            }
-	        }
-
-	        function setCheckBox2Off() {
-	            var checkbox2 = document.getElementById("checkbox2");
-	            if (checkbox2.src != "check-off.png") {
-	                checkbox2.src = "check-off.png";
-	                checkbox2.style.opacity = 0.25;
-	            }
-	        }
-	    </script>
-	</head>
-
-	<body>
-
-		<div id="instructions">
-			<div id="logo">
-				<img src="firstvoices-logo.gif" alt="FirstVoices" />
-				<p>Keyboards</p>
-			</div>
+    <head>
+        <title>FirstVoices Keyboards installation</title>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+        <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" name="viewport">
+        <link rel="stylesheet" type="text/css" href="android-instructions.css" />
+        <script language="JavaScript">
+            function showSetup() {
+                window.jsInterface.showSetup();
+            }
+
+            function showRegionList() {
+                window.jsInterface.showRegionList();
+            }
+
+            function setCheckBox1On() {
+                var checkbox1 = document.getElementById("checkbox1");
+                if (checkbox1.src != "check-on.png") {
+                    checkbox1.src = "check-on.png";
+                    checkbox1.style.opacity = 1.0;
+                }
+
+                enableButton2();
+            }
+
+            function setCheckBox1Off() {
+                var checkbox1 = document.getElementById("checkbox1");
+                if (checkbox1.src != "check-off.png") {
+                    checkbox1.src = "check-off.png";
+                    checkbox1.style.opacity = 0.25;
+                }
+
+                disableButton2();
+            }
+
+            function disableButton2() {
+                var button2 = document.getElementById("button2");
+                button2.disabled = true;
+                button2.style.opacity = 0.25;
```

Comment:
We should be able to do that with a CSS selector, e.g. `#button2:disabled { opacity: 0.25 }`